edx_rest_framework_extensions/auth/jwt/tests/test_authentication.py

```diff
@@ -235,3 +246,10 @@ def _get_test_jwt_token(self):
         payload = generate_latest_version_payload(user)
         jwt_token = generate_jwt_token(payload)
         return jwt_token
+
+
+# We want to duplicate these tests for now while we have two major code paths.  It will get unified once we have a
+# single way of doing JWT authentication again.
```

Comment:
There are some issues with this testing, but not certain how deep:
1. The final test also overrides `EDX_DRF_EXTENSIONS`, and based on codecov message, it looks like `ENABLE_FORGIVING_JWT_COOKIES` may not actually get overridden as expected. Maybe we just dupe that one test?
2. Not sure if we want to override the tests again, but:
  a. We might want a conditional around `request.META[USE_JWT_COOKIE_HEADER] = 'true'` based on `ENABLE_FORGIVING_JWT_COOKIES`, and
  b. Ideally we'd test forgiving JWTs with and without this header, to ensure it will work both before and after the header is removed. However, since you've been testing with it, if we add the conditional we'll have switched to testing without, and hopefully that is enough.

Reply:
So the existing modifications had to be made because of the case where we don't pass the header.  Previously those test were failing but with forgiving JWT auth, they begin passing so that case is already covered.

For the code coverage, I think there is some issue  with the latest coverage report being generated and sent to codecov. When I looked at the coverage locally, I was able to confirm that both code paths were getting covered after I duplicated the tests and added the override on the duplicates.

Reply:
I'm marking as resolved. I don't plan on adding a whole other set of tests, and if we rely on that header when we don't think we should, we'll find out when we go to remove it.